changelog.txt

```diff
@@ -48,6 +48,7 @@ Template for new versions:
 - `gui/pathable`: new "Depot" mode that shows whether wagons can path to your trade depot
 - `advtools`: automatically add a conversation option to "ask whereabouts of" for all your relationships (before, you could only ask whereabouts of people involved in rumors)
 - `gui/design`: all-new visually-driven UI for much improved usability
+- `makeown`: goblin snatchers and giants can now be makeown-ed more reliably
```

Comment:
needs to get moved to changelog for current version. also, should likely go in `Fixes`, and say something like "Fix enemy creatures continuing to be hostile after `makeown`"